resource/reactivedrop_italian.txt

```diff
@@ -611,8 +611,8 @@
 "rd_objective_chat_message"		"Obiettivo %s1 completato! \nTempo: %s2 Delta con obiettivo precedente: %s3\n"
 "[english]rd_points_awarding_message"		"Awarding %s4%s1 points to %s4%s2, total = %s4%s3\n"
 "rd_points_awarding_message"		"Assegnazione di %s4%s1 punti per %s4%s2, totale = %s4%s3\n"
-"[english]rd_points_reduced_due_to_farming"		"You recently played this map. Gained points will be decreased.\n"
-"rd_points_reduced_due_to_farming"		"Hai giocato questa mappa di recente. Guadagnerai meno punti.\n"
+		"[english]rd_points_reduced_due_to_farming"		"You recently played this map. Gained points have been decreased.\n"
+		"rd_points_reduced_due_to_farming"		"Avete giocato questa mappa di recente. Si guadagnano meno punti.\n"
```

Comment:
2nd person plural looks weird to me here

perhaps 2nd person singular would be better

```suggestion
		"rd_points_reduced_due_to_farming"		"Hai giocato questa mappa di recente. I punti guadagnati sono stati ridotti.\n"
```